django/db/models/base.py

```diff
@@ -370,15 +370,22 @@ def _default_manager(cls):
         return cls._meta.default_manager
 
 
+class ModelStateFieldsCacheDescriptor:
+    def __get__(self, instance, cls):
+        if instance is None:
+            return self
```

Comment:
I don't think it's really needed and if it's needed I'm not sure where to place it.